### PR TITLE
obsolete pyogenic autoinflammatory syndrome of childhood

### LINKED ISSUE #6789: [Obsolete] MONDO:0957404	pyogenic autoinflammatory syndrome of childhood
**Mondo term (ID and Label)**

MONDO:0957404 | pyogenic autoinflammatory syndrome of childhood

**Reason for deprecation**

Can't identify papers or sources that use this category
**Suggested term to consider**

Unclear since the grouping term is also being obsoleted.
**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)
https://orcid.org/0000-0001-7941-2961

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #6789

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -524968,19 +524968,18 @@ is_a: MONDO:0015137 {source="https://orcid.org/0000-0001-5208-3432"} ! periodic
...
+comment: This term's classification was reviewed in the context of the Strategic Refinement project (2023) and was determined to be excluded from the 'connective tissue disorder' (MONDO:0018153) ontology branch (https://orcid.org/0000-0002-1780-5230)
 subset: gard_rare {source="GARD:21459"}
-subset: obsoletion_candidate
 subset: orphanet_rare {source="Orphanet:324942"}
 subset: rare
 xref: GARD:21459 {source="Orphanet:324942"}
-xref: Orphanet:324942 {source="MONDO:equivalentTo"}
-is_a: MONDO:0019751 {source="https://orcid.org/0000-0001-7941-2961", source="https://orcid.org/0000-0002-5002-8648"} ! autoinflammatory syndrome
+xref: Orphanet:324942 {source="MONDO:obsoleteEquivalent"}
+property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeHistoricalDisease"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:524978]: Is this the same reason as used in the strategic refinement work? I think is should be: `property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeGrouping"}`

**Review by twhetzel — CHANGES_REQUESTED** [review]

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:524978]: this was not a grouping class, so it is not appropriate to add that exclusion reason.

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:524978]: @twhetzel could you please review again and approve?

Comment (twhetzel) [reply src/ontology/mondo-edit.obo:524978]: Oh, even though the Orphanet page says it is a grouping term: https://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=EN&Expert=324942?

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:524978]: got it! I'll add the exclusion reason

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:524978]: ready for review

**Comment by sabrinatoro** [comment]
I am questioning whether this term should be obsoleted.

**Review by sabrinatoro — APPROVED** [review]

**Comment by sabrinatoro** [comment]
@nicolevasilevsky based on the curation call (Jan 25, 2024), I have approved this PR. I also updated from Master and the QCs need to run before we can merge. 

Question for you: do you think we should keep the comment? Or should we update it to add something about the term being obsoleted?

**Comment by nicolevasilevsky** [comment]
> Question for you: do you think we should keep the comment? Or should we update it to add something about the term being obsoleted?

Personally, I think if the term is obsoleted, it is not necessary to keep the comment.

**Comment by sabrinatoro** [comment]
> Question for you: do you think we should keep the comment? Or should we update it to add something about the term being obsoleted?
> 
> Personally, I think if the term is obsoleted, it is not necessary to keep the comment.

I agree. Especially this comment makes things confusing. I will remove it. Thank you!

**Pull request merged**